Sources/Views/MessagesCollectionView.swift

```diff
@@ -64,8 +64,19 @@ open class MessagesCollectionView: UICollectionView {
     // MARK: - Methods
 
     public func scrollToBottom(animated: Bool = false) {
-        guard let indexPath = indexPathForLastItem else { return }
```

Comment:
The problem here is the indexPath is `nil` by the time this method is called. I need to find a better opportunity to call it.